class/defaults.yml

```diff
@@ -6,25 +6,19 @@ parameters:
     ignore_alerts: []
 
     charts:
-      topolvm: v4.0.0
+      topolvm:
+        source: https://topolvm.github.io/topolvm
+        version: v4.0.2
 
     images:
       topolvm:
         registry: quay.io
         repository: topolvm/topolvm-with-sidecar
-        tag: 0.10.4
+        tag: 0.10.6
 
-    deviceclasses:
-      ssd:
-        volume-group: vgssd
-        spare-gb: 10
-        default: true
+    deviceclasses: []
```

Comment:
Looks valid to me https://github.com/topolvm/topolvm/blob/main/charts/topolvm/values.yaml#L134